test/integration/branch_int_test.go

```diff
@@ -21,7 +21,7 @@ func (suite *BranchIntegrationTestSuite) TestBranch_List() {
 	ts := e2e.New(suite.T(), false)
 	defer ts.Close()
 
-	ts.PrepareProject("ActiveState-CLI/Branches", "")
+	ts.PrepareProject("ActiveState-CLI/Branches", "35af7414-b44b-4fd7-aa93-2ecad337ed2b")
```

Comment:
May I ask why this is needed? There are other uses of `ts.PrepareProject()` with empty commit IDs. The empty ID may have been intentional for the test.

Reply:
In this case the migration mechanic was breaking on it since there was nothing to migrate. And a project without a commit ID is malformed. If a test doesn't NEED it then I guess it's not an issue, but in this case it does.